### PR TITLE
re-worded the crest guide and some smaller layout fixes

### PR DESCRIPTION
**Pull request by Hootyoo**

### CODE REVIEW

File: index.rst [hunk 1]
```diff
@@ -9,6 +9,19 @@ User Documentation
...
+   cookbook/hubspotforms
+   cookbook/pageviewsandcaching
+   cookbook/requestevaluation
+   cookbook/stagingdomain
+   cookbook/rtlconversion
+   cookbook/ssl_certificates
+   cookbook/proxymodes
+   cookbook/headers
+   cookbook/securelogin
+   cookbook/sitesearch
```
Comment (zmeggyesi) [inline index.rst:24]: Should these be given a separate section in the index? The way they are now, they kinda blend together with the rest of the content...

**Review by zmeggyesi — CHANGES_REQUESTED** [review]

File: dashboard2/publishwebsite/clientsidetranslation.md [hunk 2]
```diff
@@ -6,28 +6,24 @@
...
 
 ## Publishing process
 
-The publishing method is in open beta phase and is available for testing and live use with a bit of help from our end. If you are interested in taking it for test drive, please contact our support team and we'll help you set it up. 
-
 Client-side publishing consists of two steps:
 
+### Dashboard - Publish Your Translations
+
+There are three things you must do on the Dashboard. First, head to the _Export_ section. Under _File format_, select _Client side translation (JS, export only). The rest of the options change to the correct values by default. You'll be notified via email when the process is finished. Once it arrives, go to _Previous exports_. Find the latest JavaScript export and in the action menu, select _Publish_. Finally, go to _Language selector_, select the target language(s) and click save.
```
[the remark this answers is not in the document]
Comment (zmeggyesi) [reply dashboard2/publishwebsite/clientsidetranslation.md:25]: Agreed.

File: cookbook/index.rst [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+hubspotforms
+pageviewsandcaching
+requestevaluation
+stagingdomain
+rtlconversion
+ssl_certificates
+proxymodes
+headers
+securelogin
+sitesearch
```
Comment (zmeggyesi) [inline cookbook/index.rst:17]: image
Something's fucky...

**Review by zmeggyesi — CHANGES_REQUESTED** [review]
Something's still "off" about the cookbook index...

**Comment by Hootyoo** [comment]
Kriszta said she had a few comments as well. She has a meeting with Balázs later this week to prioritise her tasks. We agreed to wait until then and work on her feedback before releasing this branch.

File: dashboard2/publishwebsite/clientsidetranslation.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (Hootyoo) [reply dashboard2/publishwebsite/clientsidetranslation.md]: [Trello card](https://trello.com/c/JECx8GVh/4899-standardise-spelling-of-hyphenated-names) created.

**Review by zmeggyesi — APPROVED** [review]

**Pull request merged**